typescript/phoenix-sdk/src/market.ts

```diff
@@ -190,11 +191,13 @@ export class Market {
     trader,
     side,
     inAmount,
+    slippage = DEFAULT_SLIPPAGE_PERCENT,
```

Comment:
docstring should have `trader` as well. any reason to use dict args here as opposed to normal args?

Reply:
I like using dict args so that order doesn't matter and the client can see exactly what the value corresponds to. in certain functions with mutilple similar args (strings, numbers, etc) it can bring some clarity to see:

{
  amount: 1,
  slippage: 0.01,
 ...
}

vs

(1, 0.01, ...)

Reply:
ok yeah cool. i think i have some comment on an old PR of @jarry-xiao's that says "why don't we use dict args" in the rust code lol.